admin-console/src/main/webapp/src/components/EmailTemplates/VIewTemplateModal.tsx

```diff
@@ -0,0 +1,30 @@
+import { ActionGroup, Button, Form, Modal, ModalVariant } from '@patternfly/react-core';
```

Comment:
Check the filename, it is `VIew` (capital I)

Reply:
The filename still shows as capital `I`. i.e. `VIew` instead of `View`

Reply:
this is not showing on my local machine, it is showing lowercase, weird.

Reply:
![Screen Shot 2022-03-29 at 9 45 24 AM](https://user-images.githubusercontent.com/86322239/160625635-c8dce930-eea5-4c53-bf06-522392ea36d2.png)